release_docs/RELEASE.txt

```diff
@@ -47,6 +47,48 @@ New Features
 
     Configuration:
     -------------
+    - Added new CMake options for building and running HDF5 API tests
+      (Experimental)
+
+      HDF5 API tests are an experimental feature primarily targeted at
+      HDF5 VOL connector authors that is currently being developed.
```

Comment:
I'm not sure about the part after "targeted at", could you look over?

Reply:
Hmm, maybe some commas would help distinguish the parts of the sentence here?

Reply:
one possibility:  targeted at authors of HDF5 VOL connectors currently being developed.

Reply:
To me that reads as though it's talking about being targeted at connectors that are currently being developed. That's true, but this is meant to say that the tests feature is currently being develop. I guess more like

"HDF5 API tests are an experimental feature, primarily targeted at HDF5 VOL connector authors, that is currently being developed."

Reply:
I have no idea whom exactly supposed to be targeted at.  Another possibility: "HDF5 API tests are an experimental feature, primarily targeted at authors, who are currently developing applications using HDF5 VOL connector"?

Reply:
Rewrite "HDF5 API tests are a work-in-progress experimental feature that is primarily aimed at HDF5 VOL connector authors."